src/Testing/ZopeTestCase/ZopeLite.py

```diff
@@ -28,14 +28,16 @@
 import time
 
 from six import exec_
+from six import PY2
 
 from Testing.ZopeTestCase import layer
 
 # Allow code to tell it is run by the test framework
 os.environ['ZOPETESTCASE'] = '1'
 
 # Increase performance on MP hardware
-sys.setcheckinterval(2500)
+if PY2:
+    sys.setcheckinterval(2500)
```

Comment:
Yeah, to be fair, this is in a test-only module and has been there forever. I've just disabled it for PY3, as that only has the switchinterval and the default value for it is generally good.